dashboard/app/helpers/levels_helper.rb

```diff
@@ -45,6 +45,10 @@ def set_videos_and_blocks_and_callouts_and_instructions
       @start_blocks = initial_blocks(current_user, @level) || @start_blocks || @level.start_blocks
     end
 
+    if current_user && params[:load_previous]
+      @start_blocks = current_user.last_attempt(@level).try(:level_source).try(:data)
```

Comment:
Prefer setting instance vars in the controller (rather than helper) but I see it's a non-trivial refactor since `@start_blocks` is set right above here in the helper.  No action needed for this PR.